client/my-sites/themes/theme-showcase.jsx

```diff
@@ -402,26 +403,62 @@ class ThemeShowcase extends Component {
 
 	renderThemes = ( themeProps ) => {
 		const tabKey = this.getSelectedTabFilter().key;
+
+		const isDiscoveryEnabled =
+			( config.isEnabled( 'themes/discovery-lits' ) && this.props.isLoggedIn ) ||
+			( config.isEnabled( 'themes/discovery-lots' ) && ! this.props.isLoggedIn );
+
 		switch ( tabKey ) {
 			case staticFilters.MYTHEMES?.key:
 				return <ThemesSelection { ...themeProps } />;
+			case staticFilters.RECOMMENDED.key:
```

Comment:
At some point, maybe when we drop the feature flag, maybe we should think about renaming RECOMMENDED. Don't think we need to change now though.